RickAndMorty-iOS/Coordinators/MainCoordinator.swift

```diff
@@ -0,0 +1,152 @@
+
+        customNavBarAppearance.largeTitleTextAttributes = [.font: UIFont(name: "Creepster-Regular", size: 34)!,
+                                                           .foregroundColor: UIColor.label,
+                                                           .shadow: shadow]
+        customNavBarAppearance.titleTextAttributes = [.font: UIFont(name: "Creepster-Regular", size: 27)!,
+                                                      .foregroundColor: UIColor.systemCyan,
+                                                      .shadow: shadow]
+
+        customNavBarAppearance.titlePositionAdjustment = UIOffset(horizontal: 0, vertical: 5)
+        customNavBarAppearance.setBackIndicatorImage(UIImage(systemName: "arrow.backward.circle"),
+                                                     transitionMaskImage: UIImage(systemName: "arrow.backward.circle"))
+
+        return customNavBarAppearance
+    }
+
+    // MARK: - Setup demo tab for experiment
+    let demoNavController = UINavigationController()
+    func setupDemo() {
+        tabBarController.addChild(demoNavController)
+        demoNavController.tabBarItem.title = "Demo"
+        let demoViewController = DemoViewController()
+        demoViewController.coordinator = self
+        demoNavController.navigationBar.backgroundColor = .systemBackground
+        demoNavController.pushViewController(demoViewController, animated: false)
+    }
+    // MARK: -
+
+    init(window: UIWindow) {
+        self.window = window
+    }
+
+    func start() {
+        // Configure tab bar.
+        tabBarController.tabBar.backgroundColor = .systemBackground
+        window.backgroundColor = .systemRed
+
+        let navBarAppearance = customNavBarAppearance()
+
+        // Add navigation controllers to the tab bar and set the title and icon for each tab.
+        tabBarController.addChild(characterNavController)
+        characterNavController.tabBarItem.image = UIImage(systemName: "person.text.rectangle")
+        characterNavController.tabBarItem.title = "Characters"
+        characterNavController.navigationBar.standardAppearance = navBarAppearance
+
+        tabBarController.addChild(locationNavController)
+        locationNavController.tabBarItem.image = UIImage(systemName: "map")
+        locationNavController.tabBarItem.title = "Locations"
+        locationNavController.navigationBar.standardAppearance = navBarAppearance
+
+        tabBarController.addChild(episodeNavController)
+        episodeNavController.tabBarItem.image = UIImage(systemName: "tv")
+        episodeNavController.tabBarItem.title = "Episodes"
+        episodeNavController.navigationBar.standardAppearance = navBarAppearance
+
+        tabBarController.addChild(searchNavController)
+        searchNavController.tabBarItem.image = UIImage(systemName: "magnifyingglass")
+        searchNavController.tabBarItem.title = "Search"
+        searchNavController.navigationBar.standardAppearance = navBarAppearance
+
```

Comment:
All resorces names should be defined as const variables as constants. Same for Strings values.
Please put it into separate file or use R.Swift